server/server.go

```diff
@@ -128,6 +131,10 @@ func NewServer(ctx *Context, stopper *util.Stopper) (*Server, error) {
 			return nil, err
 		}
 	}
+
+	//
+	s.sqlDB = sqlserver.NewDBServer(s.db)
```

Comment:
I actually like the server having its own directory. Let's leave the sqlserver package and make the full type sqlserver.Server. That would be the only type that stutters.